### ISSUE TITLE
GitHub Issues: Replace `story` with `bug` or `enhancement`

### ISSUE THREAD

**Issue by tchap** [salsaflow/salsaflow#147]
The issues recognised by the GitHub Issues module should be those labeled with `bug` or `enhancement` instead of simply `story`.

**Comment by tchap** [comment]
```
$ salsaflow story start
[RUN]      Fetch stories from the issue tracker

You can start working on one of the following stories:

  Index		Story ID	Story Title
  =====		========	===========
  1		#123		This is a bug
  2		#122		This is an enhancement

Now you can do one of the following:

  - Choose a story by inserting the associated index.
  - Press Enter or insert 'q' to abort the dialog.
  - Insert a regular expression to filter the story list.
    (case insensitive match against the story title)

Current dialog depth: 1
Choose what to do next:
```

**Comment by tchap** [comment]
Need to change this in the daemon as well before marking this as implemented.

**Comment by tchap** [comment]
Marking as implemented, see salsaflow/salsaflow-daemon#22
